spec/extend-tags-spec.js

```diff
@@ -0,0 +1,243 @@
+var tidy = require('../');
+
+describe('Extending tags', function() {
+    it('should extend opening tag up, to parent node of closing tag', function() {
+        var input = `
+            <div>
+                <div>
+                    <div>
+                        <div>
+                            Data{{#tag}}Data
+                        </div>
+                        <span>Data</span>
+                        <span>Data</span>
+                    </div>
+                    <span>Data</span>
+                    <span>Data</span>
+                </div>
+                <span>Data</span>
+                <span>Data</span>
+                {{/tag}}
+            </div>
+        `;
+
+        var result = tidy(input);
+        expect(result).toBe(
+            '<div>' +
+                '<div>' +
+                    '<div>' +
+                        '<div>' +
+                            'Data{{#tag}}Data' +
```

Comment:
@jasny Well, they are comfortable to use with html.
Do you mean why I am *not* using backquotes in expect(result).toBe(...)? Just because tabs and new lines between nodes are removed when processing template.